srcpkgs/pango/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'pango'
 pkgname=pango
-version=1.48.7
+version=1.48.9
 revision=1
 build_style=meson
 build_helper=gir
-configure_args="-Dintrospection=$(vopt_if gir enabled disabled)"
+configure_args="-Dintrospection=$(vopt_if gir enabled disabled)
+ --wrap-mode nofallback"
```

Comment:
Why did you have to add this? Might make sense as an option in the build style.

Reply:
Oh, i added to skip fallback on gi docgen locally, should have been remove it before push.